### PR TITLE
[PE-4730] Fix documentation bugs

### PR DESCRIPTION
**Pull request by mattnickles**
**Improvements**
- Added ability to toggle between Web Component and HTML Blueprint to more code examples.
- Improved JavaScript is required text within blueprint examples. Deemphasized style to avoid overpowering examples.
- Improved text on Icon doc page as this will be an entry point to Chi for many users.
- Changed progress components default color for consistency with brand.
- Made Sidenav documentation examples more consistent.

**Bug Fixes**
- Removed chi-icon.js script from documentation website and replaced all remaining svg icons in doc examples with either web component or web font icons depending on context.
- Fixed sass lint bug in icon webfont file.
- Updated old brand component code in tests and template pages with the new accessible markup.

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://14224c2ab99b2b9a303e4a6189948e547308decc-chi-pr.enterprise.ctl.io" target="_blank"> https://14224c2ab99b2b9a303e4a6189948e547308decc-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/892/ for more details.

File: src/website/views/components/forms/range-slider/index.pug [hunk 1]
```diff
@@ -3,6 +3,8 @@ title: Range Slider
 description: Range sliders are used for inputting numeric values within a range.
 tabs: {"Examples" : "examplestab", "Properties" : "propertiestab", "Accessibility" : "accessibilitytab"}
 ---
+script(type='module' src='../../../js/ce/ux-chi-ce/ux-chi-ce.esm.js')
+script(nomodule='' src='../../../js/ce/ux-chi-ce/ux-chi-ce.js')
```
[the remark this answers is not in the document]
Comment (mattnickles) [reply src/website/views/components/forms/range-slider/index.pug]: Thanks - fixed!

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://89a96fd8e7be39c0e175adcc7629f20b210b6106-chi-pr.enterprise.ctl.io" target="_blank"> https://89a96fd8e7be39c0e175adcc7629f20b210b6106-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/894/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-oclafk, https://ops.elasticbox.com/services/instances/i-tm191r

**Pull request merged**